src/Kovan-DssSpell.sol

```diff
@@ -13,73 +13,197 @@
+        address MCD_SPOT         = DssExecLib.spotter();
+        address MCD_END_OLD      = DssExecLib.end();
+        address MCD_FLIP_LINK_A  = DssExecLib.flip("LINK-A");
+        address ILK_REGISTRY_OLD = DssExecLib.reg();
+        address PIP_LINK         = DssExecLib.getChangelogAddress("PIP_LINK");
 
-    uint256 constant WAD        = 10**18;
-    uint256 constant RAD        = 10**45;
-    uint256 constant MILLION    = 10**6;
 
-    // PAXG-A
-    address constant PAXG = 0x52403FCEfcf3A810e58868fF19c34725B426473A;
-    address constant MCD_JOIN_PAXG_A = 0x822248F31bd899DE327A760a78B6C84889aF180D;
-    address constant MCD_FLIP_PAXG_A = 0x0b2e32151041641Fa37a1F54D7eD526989eF9B73;
-    address constant PIP_PAXG = 0x31CceDBc45179f17CfD34967680C6560b6509C1A;
+        // ------------------  END  ------------------
 
-    address constant PROXY_ACTIONS = 0xD8b9702755E91Aa792656966aE6bAF32F4C394Ba;
+        // Set contracts in END
+        DssExecLib.setContract(MCD_END,  "vat", MCD_VAT);
+        DssExecLib.setContract(MCD_END,  "cat", MCD_CAT);
+        DssExecLib.setContract(MCD_END,  "dog", MCD_DOG);
+        DssExecLib.setContract(MCD_END,  "vow", MCD_VOW);
+        DssExecLib.setContract(MCD_END,  "pot", MCD_POT);
+        DssExecLib.setContract(MCD_END, "spot", MCD_SPOT);
 
-    function actions() public override {
-        CollateralOpts memory PAXG_A = CollateralOpts({
-            ilk: "PAXG-A",
-            gem: PAXG,
-            join: MCD_JOIN_PAXG_A,
-            flip: MCD_FLIP_PAXG_A,
-            pip: PIP_PAXG,
-            isLiquidatable: true,
-            isOSM: true,
-            whitelistOSM: true,
-            ilkDebtCeiling: 5 * MILLION,
-            minVaultAmount: 100,
-            maxLiquidationAmount: 500,
-            liquidationPenalty: 1300,
-            ilkStabilityFee: FOUR_PCT,
-            bidIncrease: 300,
-            bidDuration: 1 hours,
-            auctionDuration: 1 hours,
-            liquidationRatio: 12500
-        });
-        addNewCollateral(PAXG_A);
-
-        DssExecLib.setChangelogAddress("PAXG", PAXG);
-        DssExecLib.setChangelogAddress("MCD_JOIN_PAXG_A", MCD_JOIN_PAXG_A);
-        DssExecLib.setChangelogAddress("MCD_FLIP_PAXG_A", MCD_FLIP_PAXG_A);
-        DssExecLib.setChangelogAddress("PIP_PAXG", PIP_PAXG);
-
-        DssExecLib.setChangelogAddress("PROXY_ACTIONS", PROXY_ACTIONS);
-
-        // Turn on Faucet
-        FaucetAbstract(DssExecLib.getChangelogAddress("FAUCET")).setAmt(PAXG, WAD);
+        // Authorize the new END in contracts
+        DssExecLib.authorize(MCD_VAT, MCD_END);
+        DssExecLib.authorize(MCD_CAT, MCD_END);
+        DssExecLib.authorize(MCD_DOG, MCD_END);
+        DssExecLib.authorize(MCD_VOW, MCD_END);
+        DssExecLib.authorize(MCD_POT, MCD_END);
+        DssExecLib.authorize(MCD_SPOT, MCD_END);
+
+        // Set wait time in END
+        DssExecLib.setEmergencyShutdownProcessingTime(EndLike(MCD_END_OLD).wait());
```

Comment:
This DssExecLib function can't be used yet since DssExecLib still thinks the end is the old end.

```suggestion
        Fileable(MCD_END).file("wait", EndLike(MCD_END_OLD).wait());
```

Reply:
Great catch!

Reply:
Fixed!